.github/actions/setup-build-env/action.yml

```diff
@@ -0,0 +1,80 @@
+name: "Setup Dependencies Variables"
+description: "Setup Dependencies Variables from other files"
+outputs:
+  CATTLE_KDM_BRANCH:
+    description: "CATTLE_KDM_BRANCH from package/Dockerfile"
+    value: ${{ steps.vars.outputs.CATTLE_KDM_BRANCH }}
+  CATTLE_K3S_VERSION:
+    description: "CATTLE_K3S_VERSION from package/Dockerfile"
+    value: ${{ steps.vars.outputs.CATTLE_K3S_VERSION }}
+  CATTLE_HELM_VERSION:
+    description: "CATTLE_HELM_VERSION from package/Dockerfile"
+    value: ${{ steps.vars.outputs.CATTLE_HELM_VERSION }}
+  HELM_VERSION:
+    description: "HELM_VERSION from package/Dockerfile"
+    value: ${{ steps.vars.outputs.HELM_VERSION }}
+  HELM_UNITTEST_VERSION:
+    description: "HELM_UNITTEST_VERSION from Dockerfile.dapper"
+    value: ${{ steps.vars.outputs.HELM_UNITTEST_VERSION }}
+  RKE_VERSION:
+    description: "RKE_VERSION from go.mod"
+    value: ${{ steps.vars.outputs.RKE_VERSION }}
+  CATTLE_RANCHER_WEBHOOK_VERSION:
+    description: "CATTLE_RANCHER_WEBHOOK_VERSION from scripts/export-config"
+    value: ${{ steps.vars.outputs.CATTLE_RANCHER_WEBHOOK_VERSION }}
+  CATTLE_CSP_ADAPTER_MIN_VERSION:
+    description: "CATTLE_CSP_ADAPTER_MIN_VERSION from scripts/export-config"
+    value: ${{ steps.vars.outputs.CATTLE_CSP_ADAPTER_MIN_VERSION }}
+  CATTLE_FLEET_VERSION:
+    description: "CATTLE_FLEET_VERSION from scripts/export-config"
+    value: ${{ steps.vars.outputs.CATTLE_FLEET_VERSION }}
```

Comment:
remove these outputs since they aren't used for `v2.7`

Reply:
Removing outputs